src/shared/components/palette/case-flag/components/manage-case-flags/manage-case-flags.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="form-group" [formGroup]="formGroup">
+  <div class="govuk-form-group" [ngClass]="{'form-group-error': errorMessages.length > 0}">
+    <fieldset class="govuk-fieldset" aria-describedby="flag-location-heading">
```

Comment:
@tarikdogan-hmcts - ideally, I would change to `flag-selection-heading`... obviously, `flag-location-heading` has come from copy/paste from @waynepHMCTS